ios/native/NativeProxy.mm

```diff
@@ -217,6 +217,25 @@ static id convertJSIValueToObjCObject(jsi::Runtime &runtime, const jsi::Value &v
     }
   };
 
+  auto configurePropsFunction = [reanimatedModule](
+                                    jsi::Runtime &rt, const jsi::Value &uiProps, const jsi::Value &nativeProps) {
+    NSMutableSet *usPropsSet = [[NSMutableSet alloc] init];
+    jsi::Array propsNames = uiProps.asObject(rt).asArray(rt);
+    for (int i = 0; i < propsNames.size(rt); i++) {
+      NSString *propName = @(propsNames.getValueAtIndex(rt, i).asString(rt).utf8(rt).c_str());
+      [usPropsSet addObject:propName];
+    }
+
+    NSMutableSet *nativePropsSet = [[NSMutableSet alloc] init];
+    propsNames = nativeProps.asObject(rt).asArray(rt);
+    for (int i = 0; i < propsNames.size(rt); i++) {
+      NSString *propName = @(propsNames.getValueAtIndex(rt, i).asString(rt).utf8(rt).c_str());
+      [nativePropsSet addObject:propName];
+    }
```

Comment:
Can we move it to a separate utility function that accepts `const jsi::Value &props` and returns `NSMutableSet *` to eliminate repetition?